build/assets/scripts/set-rps-mask.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+dev=$1
+[ -n "${dev}" ] || { echo "The device argument is missing" >&2 ; exit 1; }
+
+mask=$2
+[ -n "${mask}" ] || { echo "The mask argument is missing" >&2 ; exit 1; }
+
+dev_dir="/sys/class/net/${dev}"
+if [ ! -d "${dev_dir}" ]; then      # the net device was renamed, find the new name
```

Comment:
isn't this still racy? it should be a little safer to try the clean way first and only if the clean way fails do this fallback.
If this can't be done for any reason please just add a code comment to document it.

Reply:
Is not racy because the system renames the device or it doesn't. If the kernel decided to rename the device, it is done at kernel level way before the systemd service runs  (by observation and event flow). That was the point of the comment "the device was renamed".

Reply:
fine, then we're good to go.